app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -27,9 +27,12 @@ def perform(form526_submission_id)
 
       begin
         with_tracking(self.class.name, form526_submission.saved_claim_id, form526_submission_id) do
+          # Prefer "next" keyword here so that after-yield code continues execution
```

Comment:
It's more than a preference -- it's a bug otherwise:
```suggestion
          # Use "next" keyword here so that after-yield code continues execution and job will not have a `try` status
```

Reply:
Here are some solutions to remedy the repetition: https://gist.github.com/myronmarston/2377461